pkg/config/config.go

```diff
@@ -30,9 +31,32 @@ type Config struct {
 	Checks       ChecksConfig  `json:"checks,omitempty" mapstructure:"checks"`
 }
 
+// Returns a list of default config filenames to check if parameter isn't passed in
+func getDefaultConfigFilenames() []string {
+	return []string{".kube-linter.yaml", ".kube-linter.yml", ".kube-linter"}
+}
```

Comment:
The reason for this is you cannot define a constant slice in Go (though correct me if I'm wrong), so instead I return a slice that has the same elements every time. If there is a better way of doing so, please let me know.

Reply:
Go does not, but since this is an unexported variable, we can assume that all users (which must be in this package) will do the right thing and not mutate it. This is the general convention we've used for "constant" slices -- use a function if it's exported, a `var` if it's not. 

For _some_ compile-time proection, you can make it an array by defining it as:
```go
var (
   defaultConfigFilenames = [...]string{".kube-linter.yaml", ".kube-linter.yaml"}
)
```

This will prevent appends, although it will not prevent mutations (like `defaultConfigFilenames[0] = "blah"`).